src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -403,21 +403,18 @@ public void testAddAction() throws Exception{
         
         List<String> expectedStatusStrings = new ArrayList<String>();
         expectedStatusStrings.add(String.format(
-                FieldValidator.TIME_FRAME_ERROR_MESSAGE,
+                FieldValidator.TIME_FRAME_ERROR_MESSAGE_WITHOUT_HTML,
```

Comment:
Again, error message without HTML are needed for testing purposes because `WebElement.getText()` strips the `<b></b>` tags